package.json

```diff
@@ -2,9 +2,9 @@
   "name": "@balancer/frontend-monorepo",
   "private": true,
   "scripts": {
-    "build": "turbo build",
-    "build:dev": "turbo build:dev",
-    "dev": "turbo dev",
+    "build": "turbo build --filter frontend-v3",
+    "build:dev": "turbo build:dev --filter frontend-v3",
+    "dev": "turbo dev --filter frontend-v3",
```

Comment:
as this will only be used locally i think we can filter here (or just the 'dev' task)

Reply:
Like discussed, please don't do this for now. Feel free to add dev:bal and dev:beets to turbo.json to run specific apps locally.

Reply:
Updated this.